python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_categorical.py

```diff
@@ -0,0 +1,375 @@
+"""
+Kriging: metamodel with continuous and categorical variables
+============================================================
+"""
+# %%
+# We consider here the surrogate modeling of an analytical function characterized by
+# continuous and categorical variables
+#
+
+# %%
+import openturns as ot
+import openturns.experimental as otexp
+import numpy as np
+import matplotlib.pyplot as plt
+
+ot.RandomGenerator.SetSeed(5)
```

Comment:
When the library is loaded, the seed is automatically set to zero. Setting the seed to 5 is either a choice that is unnecessary (because removing it generates an interesting example) or deserves a comment to explain its purpose.